lib/compress.js

```diff
@@ -1053,7 +1053,7 @@ merge(Compressor.prototype, {
             var val = {};
             for (var i = this.properties.length; --i >= 0;) {
                 var prop = this.properties[i];
-                if (prop instanceof AST_ObjectKeyVal) {
+                if (typeof prop.key === "string") {
```

Comment:
I don't think that's advisable because it could skip a side effect for key/value in a computed prop or a situation where there's a non-string literal key.